pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/AbstractIntegrationTest.java

```diff
@@ -44,6 +46,14 @@ public abstract class AbstractIntegrationTest {
 
     String mavenInstallationName;
 
+    public static GenericContainer<?> createContainer(String target) {
+        return new GenericContainer<>(new ImageFromDockerfile(
+                                "localhost/pipeline-maven/" + target, Boolean.parseBoolean(System.getenv("CI")))
```

Comment:
Or this could be set to unconditional `false`, if there is still a concern about CI concurrently running tests which throw away each others’ layer caches.